walsamlee/politico
# #163416831 Add Delete Party by ID Endpoint

#### What does this PR do?
Add `delete-party-by-ID` API endpoint
#### Description of Task to be completed?
Have `/api/v1/parties/:partyId` endpoint working
#### How should this be manually tested?
- Clone the project, https://github.com/walsamlee/politico
- Check out into the proper branch, `ft-v1-delete-party-163416831`
- Using postman, `DELETE` with API endpoint URL `/api/v1/parties/:partyId` 
#### What are the relevant pivotal tracker stories?
#163416831
